pkg/rancher-desktop/config/settingsImpl.ts

```diff
@@ -470,6 +470,12 @@ export const updateTable: Record<number, (settings: any, locked : boolean) => vo
     }
   },
   10: (settings, locked) => {
+    // virtiofs is not supported in Rancher Desktop 1.13.0
+    if (!_.isEmpty(_.get(settings, 'experimental.virtualMachine.mount.type'))) {
```

Comment:
Did you want `_.isSet`?
Or just do `if (settings.experimental?.virtualMachine?.mount?.type ===`

Reply:
Jan says this is the same as a previous one; good enough.